tutorials/introductory/usage.py

```diff
@@ -121,8 +121,8 @@
 # =====================================
 #
 # All of plotting functions expect `numpy.array` or `numpy.ma.masked_array` as
-# input.  Classes that are 'array-like' such as `pandas` data objects
-# and `numpy.matrix` may or may not work as intended.  It is best to
+# input.  Classes that are similar to arrays such as `pandas` data objects
+# and `numpy.matrix` may not work as intended.  Common convention is to
```

Comment:
You can remove ` numpy.matrix`. That class is not recommended anymore by numpy itself, and rarely used in practice nowadays. - Also the matrix example code below.

Reply:
I think this is a bit too strong, we do test that pandas works correctly in some cases and special case extracting the index and label when it makes sense.